src/common/dao/replication_job.go

```diff
@@ -136,6 +137,114 @@ func AddRepPolicy(policy models.RepPolicy) (int64, error) {
 	return policyID, nil
 }
 
+// AddRepPolicyNg ...
+func AddRepPolicyNg(policy models.RepPolicyNg) (int64, error) {
+	o := GetOrmer()
+	sql := `insert into replication_policy_ng (name, description, creator, src_registry_id, src_namespaces,
```

Comment:
Use the function provided by the orm rather than the raw SQL

Reply:
I'll remove raw sql with ORM.But why you use raw sql before?